src/llama_stack/providers/registry/agents.py

```diff
@@ -38,4 +38,23 @@ def available_providers() -> list[ProviderSpec]:
             ],
             description="Meta's reference implementation of an agent system that can use tools, access vector databases, and perform complex reasoning tasks.",
         ),
+        InlineProviderSpec(
+            api=Api.agents,
+            provider_type="inline::dana",
```

Comment:
`inline` means the agent code will be contributed directly to llama stack and run within the stack server process.

`remote` means the agent will be running outside the stack (e.g. dana deploy) and adapter code will be present within stack.

is your intention to implement the agent directly within stack or have stack call out to the agent?

Reply:
It'll be inline, using the Dana library. Dana-side we also are making usage adapters for other LlamaStack APIs.